gammapy/cube/make.py

```diff
@@ -268,9 +275,30 @@ def run(self, selection=None):
         for name in selection:
             getattr(self, "_make_" + name)()
 
+        background_model = BackgroundModel(self.maps["background"])
+
+        if "psf" in self.maps:
```

Comment:
As `self.maps` is a dictionary the `if` and `else` blocks could be removed and replaced by `psf = self.maps.get("psf")`